cloudmarker/stores/emailstore.py

```diff
@@ -0,0 +1,68 @@
+"""email store plugin."""
+
+
+import logging
+from email.mime.multipart import MIMEMultipart
+from email.utils import COMMASPACE, formatdate
+from smtplib import SMTP, SMTPException
+
+# Define module-level logger.
+_log = logging.getLogger(__name__)
+
+
+class EmailStore():
+    """A plugin to send email notification for anomalies found."""
+
+    def __init__(self, host, port, subject, to, sender, body):
+        """Create an instance of :class:`EmailStore` plugin.
+
+        Arguments:
+            host (string): Server hostname for SMTP.
+            port (int): Server port for STMP.
+            subject (string): Subject line for email.
+            to (list): List of recipients(string).
+            sender (string): From field for email.
+            body (string): Email body.
+
+        """
+        self.host = host
+        self.port = port
+        self.subject = subject
+        self.to = to
+        self.sender = sender
+        self.body = body
+
+    def write(self, record):
+        """Add record to a buffer to sent via email.
+
+        This method does nothing for now. However, this method
+        show keep adding all the records to the buffer to
+        send via email to particular recipient.
+        """
+
+    def done(self):
+        """Perform final cleanup tasks.
+
+        This method is called after all records have been written. In
+        this example implementation, we properly terminate the JSON
+        array in the .tmp file. Then we rename the .tmp file to .json
+        file.
+
+        Note that other implementations of a store may perform tasks
+        like closing a connection to a remote store or flushing any
+        remaining records in a buffer.
+
+        """
+        message = MIMEMultipart()
+        message['Date'] = formatdate(localtime=True)
```

Comment:
This will get changed to:
```python
message['Date'] = email.utils.formatdate(localtime=True)
```

Reply:
Currently, I am changing this as asked to, but I disagree which the approach of importing full library. We can have a discussion on this.

Reply:
I haven't understood the disagreement entirely. If we want to use an identifier from another module, we have to import the full module anyway. There is no way to avoid that. That's just how Python is designed.

It appears to me that the disagreement is not regarding whether to import a full module or not but about the coding style on whether to write fully qualified identifiers or unqualified identifiers.